app/routes.py

```diff
@@ -1,2 +1,212 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import request, Blueprint, make_response, jsonify
+from datetime import datetime
+import requests
 
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+def customer_response_helper(customer):
```

Comment:
👍 yay helper functions! one thing to think about in the future is any functions like this should be put in the models. the routes should be very bare bones. Routes should only query the database, call model methods, and direct the return statement. Everything else, thinkaboutt putting into models or a separate helper file!